assets/js/windup.js

```diff
@@ -1,5 +1,5 @@
 $(".windup-download-now").click(function() {
-    window.location = 'https://www.jboss.org/download-manager/file/jboss-migrationtoolkit-windup-2.4.0.GA.zip'
+    window.location = 'https://www.jboss.org/download-manager/file/windup-distribution-2.5.0.Final-offline.zip'
```

Comment:
Rodney mentioned we should have this file named "jboss-migrationtoolkit-windup-2.5.0.GA.zip" on the Download Manager, same as for 2.4. It's the same file, just renamed. I posted a comment on the JIRA to ask for that update.